Assignment/Complex.h

```diff
@@ -0,0 +1,31 @@
+// ------------------------------------------------------------------------------------------------
+// Training ~ A training program for new joiners at Metamation, Batch - July 2024.
+// Copyright (c) Metamation India.
+// Vinitha Perumal - GET
+// ------------------------------------------------------------------
+// Complex.h
+// Program on Test1.2 branch.
+// ------------------------------------------------------------------------------------------------
+#ifndef COMPLEX_H
+#define COMPLEX_H
+
+typedef struct ComplexNumber {
+   float real, img;

```

Comment:
```suggestion
   float Real, Img;

```
Please follow naming conventions. Previously given reviews are valid for every following assignment